docker/Debug.dockerfile

```diff
@@ -4,7 +4,7 @@
 # Licensed under the MIT license. See LICENSE.txt in the project root for license information.
 
 # Based on latest image cached by Azure Pipelines: https://docs.microsoft.com/azure/devops/pipelines/agents/hosted#software
-FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019
+FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2022
```

Comment:
Why `aspnet`? Should be SDK; though, I suppose we don't actually need the full SDK. In that case, probably just `runtime`. Still, given the size of Windows containers, a small difference between `runtime` and `sdk` probably isn't worth the hassle if we ever need anything from `sdk`.

Reply:
Actually, we probably will want `sdk` on this one and `runtime` on `docker/Dockerfile`.

Reply:
No worries, I'll change this. I just copied over what we did for the VSSetup.Powershell change assuming we needed the same image since that went from sdk->aspnet.

Reply:
Who the heck did that over there?!

`<img src="Homer-Simpson-backing-into-the-bushes.gif" />`

Reply:
You might fix it over there, or I can. Sorry about that.